.vitepress/constants/constants.js

```diff
@@ -1,19 +1,25 @@
 const constants = Object.freeze({
   golangVersion: "go1.22.3",
 
-  rollkitLatestTag: "v1.0.0-beta.1",
-  rollkitLatestSha: "cd1970de",
-  rollkitCosmosSDKVersion: "v0.50.6-rollkit-v0.13.3-no-fraud-proofs",
-  rollkitIgniteAppVersion: "rollkit/v0.2.1",
+  evolveLatestTag: "v1.0.0-beta.1",
+  evolveLatestSha: "cd1970de",
+  evolveCosmosSDKVersion: "v0.50.6-evolve-v0.13.3-no-fraud-proofs",
+  evolveIgniteAppVersion: "evolve/v0.2.1",
```

Comment:
rollkit/v0.3.0

a new tag will happen once we are properly updated (ev-node / ev-abci, etc..): https://github.com/ignite/apps/pull/220